### PR TITLE
Create BNFabb96.xml

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New record for Abb 96, a complex and tricky manuscript.

### CODE REVIEW

File: ParisBNF/abb/BNFabb96.xml [hunk 1]
```diff
@@ -0,0 +1,675 @@
...
+                                     <note><ref target="#p4">Unit 4</ref> is composed of only
+                                        one small rectangular parchment leaf mounted
+                                        on a stub.</note>
+                                  </collation>
+                               </supportDesc>
+
+                               <layoutDesc>
+                                  <layout columns="1" writtenLines="4">
+                                       <locus target="#84r"/>
+                                      <dimensions unit="mm" xml:id="textarea3">
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb96.xml:568]: this can be deleted, if no dimensions are available

File: ParisBNF/abb/BNFabb96.xml [hunk 2]
```diff
@@ -0,0 +1,675 @@
...
+               <term key="ChristianLiterature"/>
+               <term key="Asmat"/>
+                <term key="Chronography"/>
+                 <term key="Translation"/>
+                    <term key="Theology"/>
+               <term key="Paks2"/>
+            </keywords>
+         </textClass>
+         <langUsage><language ident="en">English</language>
+         <language ident="cop">Coptic Bohairic</language>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb96.xml:662]: Where does this manuscript contain coptic?

**Review by thea-m — CHANGES_REQUESTED** [review]
Yes, very complex!

File: ParisBNF/abb/BNFabb96.xml [hunk 3]
```diff
@@ -0,0 +1,675 @@
...
+
+                    <physDesc>
+                       <objectDesc form="Codex">
+                          <supportDesc>
+                             <support>
+                                <material key="parchment"/>
+                             </support>
+                             <extent>
+                               <measure unit="leaf">151</measure>
+                               <measure unit="quire"></measure>
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb96.xml:42]: ok

File: ParisBNF/abb/BNFabb96.xml [hunk 4]
```diff
@@ -0,0 +1,675 @@
...
+
+                         <item xml:id="e4">
+                            <locus target="#146r"/>
+                            <desc type="Unclear">Partly unintelligeble short note, written in a crude hand</desc>
+                            </item>
+
+                  <item xml:id="e5">
+                    <locus target="#151v"></locus>
+                  <desc type="StampExlibris">
+                     Shelfmark of the Library on the spine</desc>
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb96.xml:324]: No. Deleted

File: ParisBNF/abb/BNFabb96.xml [hunk 5]
```diff
@@ -0,0 +1,675 @@
...
+                              ቀዳማይ፡ ነቢይ፡ ኢዮብ፡ ተነበየ፡ ፵ዓመት፨ ወእምድህሬሁ፡ ሙሴ፡ ተነበየ፡ ፵ዓመት፡ ቅድመ፡
+                              ምጽኣተ፡ እግዚእነ፡ በሥጋ፨ ፲ወ፬፻፹ወ፭ዓመት፨ ወእምሕሬሁ፡ ኢያሱ፡ ዘኔዌ፡ እስከ፡ ፳ወ፯፳ወ፯ዓመት፨
+                              </incipit>
+                              <note></note>
+                           </msItem>
+
+                              <msItem xml:id="p2_i3">
+                                  <locus from="148r" to="148v"/>
+                                 <title type="complete" ref="LIT2293Sawase">A short
+                                   sawāsǝw-lexicon for</title>
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb96.xml:391]: Es war ein Satz mit Kodierung, die irgendwie nicht durchkam?

**Review by thea-m — APPROVED** [review]

**Pull request merged**